homeassistant/components/input_number.py

```diff
@@ -1,8 +1,8 @@
 """
```

Comment:
Please revert the change in file permission.

Reply:
Oops. I try to catch it. It's an artifact of doing all my Git work on a Linux VM but working on the files via SMB share on my Windows PC in Visual Studio... every time I save I have to both force Unix line endings in Visual Studio and then afterwards remove the execute bit ... 

I'll fix this when I fix rename the service call.